x-pack/plugins/reporting/public/components/reporting_panel_content.tsx

```diff
@@ -129,7 +139,10 @@ class ReportingPanelContentUi extends Component<Props, State> {
         </EuiText>
         <EuiSpacer size="s" />
 
-        <EuiCopy textToCopy={this.state.absoluteUrl} anchorClassName="eui-displayBlock">
+        <EuiCopy
```

Comment:
This needs to be massaged a bit more, as there's some single quotes that aren't properly escaped.